projects/1.18/assets/modular-routers/modularrouters/lang/zh_cn.json

```
@@ -1 +1,357 @@
+  "modularrouters.itemText.usage.item.extruder_module_2": "与造桥模块1型类似，但是会按照内部的模板栏设定的方块来放置模板方块而不是放置路由器缓冲区中的真实方块。",
+  "modularrouters.itemText.usage.item.fast_pickup_augment": "允许虚空模块无视一般情况下捡起掉落物的延迟时间。",
+  "modularrouters.itemText.usage.item.filter_round_robin_augment" : "不会执行默认的任何/全部匹配。路由器会根据过滤器在每个执行刻匹配单个物品。所有被选中的物品会进行递增循环。${br}这对处理一系列含有特殊顺序的物品很有用。",
+  "modularrouters.itemText.usage.item.flinger_module": "从路由器的缓冲区中将物品长距离投掷到世界。 可以在模块的GUI中调整投掷的速度与距离，仰角与偏角。",
+  "modularrouters.itemText.usage.item.fluid_module_2": "从任何相邻方块里转移液体到路由器或者从路由器到方块。 ${br}• 路由器的缓冲区中必须装有流体容器。${br}• 可以和储罐或世界中转换液体。",
+  "modularrouters.itemText.usage.item.fluid_module": "在模块设置的方向上将流体传输到路由器或向模块的方向传输流体。${br}• 路由器缓冲区中必须有流体容器。${br}• 流体可以通过设置从容器放置到世界或者相反。",
+  "modularrouters.itemText.usage.item.fluid_upgrade": "增加路由器可以传输（在每个方向）上的%d mB/刻单位的流体量。",
+  "modularrouters.itemText.usage.item.fluidUpgradeRouter": "• 此路由器的传输速率：%d mB/刻",
+  "modularrouters.itemText.usage.item.inspection_filter": "通过检查物品的各种属性过滤物品。可以像任何物品一样录入到路由器过滤名单中。${br}• 右击：查看/配置。",
+  "modularrouters.itemText.usage.item.mimic_augment": "使造桥器模块2型产生的虚拟方块同时模拟原方块的亮度、红石充能、硬度和爆炸抗性。",
+  "modularrouters.itemText.usage.item.mod_filter": "按照一个物品来自的模组ID过滤物品，可以像任何物品一样录入到路由器过滤名单中。${br}• 右击：查看/配置。",
+  "modularrouters.itemText.usage.item.muffler_upgrade": "抑制来自路由器/模块的声音和粒子效果：${br}• 1个升级会抑制声音效果${br}• 2个或更多个会抑制更多粒子效果${br}• 3个或更多个则会抑制路由器的运行动画。",
+  "modularrouters.itemText.usage.item.pickup_delay_augment": "每个拓展会增加10刻捡起掉落物品的延迟时间。",
+  "modularrouters.itemText.usage.item.placer_module": "如果可能，将路由器缓冲区中的方块或物品放置到模块配置的方向。",
+  "modularrouters.itemText.usage.item.player_module": "在玩家的个人存储列表和路由器的缓冲区之间远程传输物品。",
+  "modularrouters.itemText.usage.item.puller_module_1": "从模块已配置方向的相邻容器中将物品提取到路由器的缓冲区中。",
+  "modularrouters.itemText.usage.item.puller_module_2": "从附近的任何存储容器中将物品提取到路由器缓冲区中。",
+  "modularrouters.itemText.usage.item.pushing_augment": "增加造桥器1型/2型伸出时推动实体的力量。可堆叠。",
+  "modularrouters.itemText.usage.item.range_down_augment": "每个拓展会将模块的作用范围减少1格。",
+  "modularrouters.itemText.usage.item.range_up_augment": "每个拓展会将模块的作用范围增加1格（直到达到模块提示信息中描述的最大值）",
+  "modularrouters.itemText.usage.item.redstone_augment": "允许模块指定自身的红石特性，这会无视路由器的红石特性。",
+  "modularrouters.itemText.usage.item.regex_filter": "通过将其显示名称与一个或多个正则表达式匹配来过滤物品。可以像任何物品一样录入到路由器过滤名单中。${br}• 右击：查看/配置。",
+  "modularrouters.itemText.usage.item.regulator_augment": "允许预先设置在容器中保留多少物品。",
+  "modularrouters.itemText.usage.item.security_upgrade": "保护路由器; 只有列出的玩家可以访问路由器。${br}• 右击玩家以记录到模块${br}• 潜行+右击玩家以删除他们的配置",
+  "modularrouters.itemText.usage.item.sender_module_1": "将路由器缓冲区中的物品直接沿X/Y/Z轴发送到选定的容器里。${br}不能有方块阻隔；不透明的方块块将阻止发送。",
+  "modularrouters.itemText.usage.item.sender_module_2": "将路由器缓冲区中的物品发送到临近的容器里。${br}传输可以穿透方块。",
+  "modularrouters.itemText.usage.item.sender_module_3": "可以将路由器缓冲区中的物品发送到任何维度的任何容器内。",
+  "modularrouters.itemText.usage.item.speed_upgrade": "提高路由器工作间隔的速度。${br}• 路由器每隔%.02f秒（%d刻）进行一次工作${br}• 升级最多可以放置：%d个",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "modularrouters.itemText.usage.item.speed_upgrade": "提高路由器工作间隔的速度。${br}• 路由器每隔%.02f秒（%d刻）工作一次${br}• 升级最多可以放置：%d个",
```
原文：**Increases the router tick rate.${br}• This router ticks every %.02f sec (%d ticks)${br}• Max. useful upgrades: %d**